src/main/java/me/kixstar/dungeons/dungeons/ChestUtils.java

```diff
@@ -10,14 +12,24 @@
 public class ChestUtils {
 
     private static final Random random = new Random();
-
     /**
      * distributeItems distributes a list of items into different chests, randomly
      * @param chests
      * @param items
      */
     static void distributeItems(List<Chest> chests, List<ItemStack> items) {
-
+        if (chests != null && items != null && !(items.isEmpty())) {
```

Comment:
I'd vastly prefer is we adopt the mentality of early exit conditions, over nesting if statements to improve readability and understanding of the code. So instead of doing 

if(xy and z is true) {
     DoTheThing();
}

Go with something like this

if(xy and z not true) {
   return;
}

DoTheThing();

It helps keeping nesting of code to a minimum.